include/lsst/jointcal/AstroUtils.h

```diff
@@ -1,37 +1,15 @@
 // This may look like C code, but it is really -*- C++ -*-
```

Comment:
Should be [parseable by emacs](https://developer.lsst.io/coding/cpp_style_guide.html#a-the-head-of-each-h-and-each-cc-file-must-begin-with-a-fixed-format-comment-declaring-the-file-contents-and-format-to-emacs).